docker-compose.yml

```diff
@@ -0,0 +1,67 @@
+version: "3.7"
+services:
+  app:
+    depends_on:
+    - db
+    build:
+      args:
+        user: www
+        uid: 1000
+      context: ./
+      dockerfile: development/Dockerfile
+    image: laravel-image
+    container_name: laravel-app
+    restart: unless-stopped
+    working_dir: /var/www/
+    volumes:
+      - projectroot:/var/www
+      - ./development/php/local.ini:/usr/local/etc/php/conf.d/local.ini
+      - ~/.composer/auth.json:/home/www/.composer/auth.json
+    networks:
+      - app-network
+
+  db:
+    image: mysql:5.7
+    container_name: laravel-db
+    restart: unless-stopped
+    ports:
+      - "3306:3306"
+    environment:
+      MYSQL_DATABASE: ${DB_DATABASE}
+      MYSQL_ROOT_PASSWORD: ${DB_PASSWORD}
+      MYSQL_PASSWORD: ${DB_PASSWORD}
+      MYSQL_USER: ${DB_USERNAME}
+      SERVICE_TAGS: dev
+      SERVICE_NAME: mysql
+    volumes:
+      - ./development/mysql/my.cnf:/etc/mysql/my.cnf
+      - ./development/mysql/docker-entrypoint-initdb.d/:/docker-entrypoint-initdb.d/
+    networks:
+      - app-network
+
+  nginx:
+    image: nginx:alpine
+    container_name: laravel-nginx
+    restart: unless-stopped
+    ports:
+      - 8000:80
+    volumes:
+      - projectroot:/var/www
+      - ./development/nginx/conf.d/:/etc/nginx/conf.d/
+    networks:
+      - app-network
+
+networks:
+  app-network:
+    driver: bridge
+
+volumes:
+  projectroot:
+    driver: local
+    driver_opts:
+      type: nfs
+      o: addr=host.docker.internal,rw,nolock,hard,nointr,nfsvers=3
+      # Certain versions of Catalina / Docker might need to you to use
+      # this instead, to fix permission errors:
+      # device: ":/System/Volumes/Data${PWD}
+      device: ":${PWD}"
```

Comment:
This will not work for Linux devs. If we want to include NFS support we probably should do it [like in `reload/ding-docker` and suggest copying over a file](https://github.com/reload/ding-docker#using-nfs-with-docker-for-mac).

Reply:
yes